upstream issues 1560
https://github.com/Supervisor/supervisor/pull/1675.patch
upstream will not merge soon as it's py3 only (find_namespace_packages)
--- a/setup.py
+++ b/setup.py
@@ -34,7 +34,7 @@
     'pytest-cov',
     ]
 
-from setuptools import setup, find_packages
+from setuptools import setup, find_namespace_packages
 here = os.path.abspath(os.path.dirname(__file__))
 try:
     with open(os.path.join(here, 'README.rst'), 'r') as f:
@@ -91,15 +91,19 @@
     classifiers=CLASSIFIERS,
     author="Chris McDonough",
     author_email="chrism@plope.com",
-    packages=find_packages(),
+    packages=find_namespace_packages(exclude=["docs","supervisor.tests","supervisor.tests.*"]),
+    package_dir={"": "."},
+    package_data={
+        "supervisor": ["version.txt"],
+        "supervisor.ui": ["**/*"],
+        "supervisor.skel": ["*"]
+    },
     install_requires=requires,
     extras_require={
         'testing': testing_extras,
     },
-    tests_require=tests_require,
     include_package_data=True,
     zip_safe=False,
-    test_suite="supervisor.tests",
     entry_points={
         'console_scripts': [
             'supervisord = supervisor.supervisord:main',
--- a/setup.cfg
+++ b/setup.cfg
@@ -2,4 +2,4 @@
 zip_ok = false
 
 [bdist_wheel]
-universal = 1
+universal = 0
